subsys/sd/sd_ops.c

```diff
@@ -214,6 +214,7 @@ static int sdmmc_spi_read_cxd(struct sd_card *card, uint32_t opcode, uint32_t *c
 	ret = sdhc_request(card->sdhc, &cmd, &data);
 	if (ret) {
 		LOG_DBG("CMD%d failed: %d", opcode, ret);
+		return ret;
```

Comment:
If it is changed to be handled as an error, then the logging message above should also be changed to LOG_ERR().

Reply:
It's not in the function below either, but I do agree.

Reply:
I can remove this again if needed, as we really shouldn't get to this point with an invalid SD card.

Reply:
Removed, to not muddle the PR